scarb/src/bin/scarb/commands/proc_macro_server.rs

```diff
@@ -0,0 +1,7 @@
+use anyhow::Result;
+use scarb::core::Config;
+
+#[tracing::instrument(skip_all, level = "info")]
+pub fn run(_config: &Config) -> Result<()> {
+    unimplemented!()
```

Comment:
total nitpick: that's the proper macro to use here 😛 

```suggestion
    todo!()
```